src/clj/game/cards/events.clj

```diff
@@ -522,6 +522,32 @@
     :msg "look at the top 4 cards of their Stack and add 1 of them to their Grip"
     :effect (effect (move target :hand) (shuffle! :deck))}
 
+   "Falsified Credentials"
+   {:prompt "Choose a type"
+    :choices ["Agenda" "Asset" "Upgrade"]
```

Comment:
So we don't currently use delayed completion with the effects of runner cards (only their `:abilities` handlers), because there are no Runner equivalents to AccDiagnostics / Subcontract that exect two card effects in a row and need to wait for the first to finish before the next goes. However, it's not a bad idea to start future proofing runner Events with delayed completion when appropriate (it's also good practice for learning when the system needs to be used). Because this card involves a delay-able effect (expose, which triggers optional abilities like prevention), its root ability must be marked `:delayed-completion true`.